Sources/Mini/Utils/OrderedSet.swift

```diff
@@ -159,7 +159,13 @@ public class OrderedSet<T: Comparable> {
 
     /// For each function
     public func forEach(_ body: (T) -> Swift.Void) {
-        return internalSet.forEach(body)
```

Comment:
why do you use NSArray?

Reply:
Only `NSArray` has the functionality to make a concurrent access to its items, speeding up the `Action` spreading.